packages/jest-jasmine2/src/reporter.ts

```diff
@@ -113,15 +113,19 @@ export default class Jasmine2Reporter implements Reporter {
     return this._resultsPromise;
   }
 
-  private _addMissingMessageToStack(stack: string, message?: string) {
+  private _addMissingMessageToStack(stack: unknown, message?: string) {
```

Comment:
I think this should also be reverted, and the check here improved instead: https://github.com/facebook/jest/blob/7b1568d9162fafe6a630fb425f16a44998894164/packages/jest-jasmine2/src/reporter.ts#L154-L157

Reply:
Yes, I agree.